pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -0,0 +1,175 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package nodelabeller
+
+import (
+	"os"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/testutils"
+)
+
+var _ = Describe("Node-labeller config", func() {
+	var nlController *NodeLabeller
```

Comment:
we need something to validate the `run()` function works as expected and makes the expected API calls to the apiserver.

something like this, where we do a controlled execution of the reconcile loop and validate the Patch command gives us the expected results. 
https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-controller/watch/node_test.go#L230